projects/uitpas/docs/creating-rewards.md

```diff
@@ -0,0 +1,75 @@
+# Creating rewards
+
+There are two ways to create new rewards:
+
+*   The UiTPAS regional coordinator can create new rewards via the UiTPAS admin tools.
+*   New rewards can be created programmatically via the API.
+
+This guide explains **how to create UiTPAS rewards programmatically**.
+
+To learn how to let passholders pay with UiTPAS points in your application, [check out this guide](pay-with-UiTPAS-points.md).
+
+## Before you start
+
+Before starting this guide, make sure you clearly understand which type of reward you want to create: [a **welcome reward** or a **points reward**](terminology.md#rewards).
+
+## Authentication
+
+Before you can [create rewards](/reference/uitpas.json/paths/~1rewards/post), you'll need **client credentials**, so you can access the UiTPAS API using a [Client Access Token](https://publiq.stoplight.io/docs/authentication/docs/client-access-token.md) or [User access Token](https://publiq.stoplight.io/docs/authentication/docs/user-access-token.md).
+
+To decide what kind of token to use, see the [overview of token types](https://publiq.stoplight.io/docs/authentication/docs/methods.md).
+
+> ##### Permissions to create and redeem rewards
+>
+> To be able to create and redeem rewards for an organizer, your client needs to receive the related rewards permissions for the organizer you are creating and redeeming these rewards for.
+
+## How to create rewards programmatically
+
+Rewards are published to public channels, such as [uitpas.be](https://www.uitpas.be). That's why it's important to clearly describe your reward and add as much detail as possible.
+
+To create a reward, **the following parameters are required:**
```

Comment:
```suggestion
To create a reward, **the following fields are required:**
```
technically, a create is happens using a request body post, which has fields, not parameters